node/tests/k8st/tests/test_health.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2018-2019 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import re
+from subprocess import CalledProcessError
+
+from tests.k8st.test_base import TestBase
+from tests.k8st.utils.utils import DiagsCollector, calicoctl, kubectl, exec_in_calico_node, run, retry_until_success, node_info, start_external_node_with_bgp, update_ds_env
+
+_log = logging.getLogger(__name__)
+_log.setLevel(logging.DEBUG)
+
+
+class TestReadiness(TestBase):
```

Comment:
@caseydavenport is the expectation that this new test logic written in Python will be able to live with the remaining Python tests for as long as we want to keep them around? 

Meaning, there isn't any reason in the near future to move these over to Golang? 

Was the reason to leave them in Python form simply a matter of being pragmatic? I.e. it would have been more overhead to convert them over because of surrounding utility functions etc (or that it makes more sense to be grouped with other existing Python tests).